### PR TITLE
Update protocol params

### PR DESCRIPTION
**Pull request by thibault-martinez**
https://github.com/iotaledger/tips-draft/pull/25/commits/6a7ab0bd34dadb89b437cde2d9e5c59ab13cf3a9

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

**Review by abdulmth — DISMISSED** [review]

**Review by kwek20 — DISMISSED** [review]

File: sdk/src/types/block/protocol.rs [hunk 1]
```diff
@@ -180,46 +184,46 @@ impl ProtocolParameters {
 #[getset(get_copy = "pub")]
 pub struct WorkScoreStructure {
     /// Modifier for network traffic per byte.
-    data_kilobyte: u32,
+    work_score_data_kilobyte: u32,
```
Comment (Alex6323) [inline sdk/src/types/block/protocol.rs]: I think this is going to change to `kibibyte`. Also I asked whether we should drop the `work_score` prefix in the TIP.

Comment (DaughterOfMars) [reply sdk/src/types/block/protocol.rs]: kibiwhatnow

Comment (thibault-martinez) [reply sdk/src/types/block/protocol.rs]: It's been changed

**Review by kwek20 — APPROVED** [review]

File: sdk/src/types/block/protocol.rs [hunk 2]
```diff
@@ -180,7 +184,7 @@ impl ProtocolParameters {
 #[getset(get_copy = "pub")]
 pub struct WorkScoreStructure {
     /// Modifier for network traffic per byte.
-    data_kilobyte: u32,
+    data_byte: u32,
```
Comment (abdulmth) [inline sdk/src/types/block/protocol.rs:187]: is this byte or kibibyte? in TIP it's kibibyte, no?

Comment (thibault-martinez) [reply sdk/src/types/block/protocol.rs:187]: It's byte, it'll be changed in the TIP

**Review by abdulmth — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**